### PR TITLE
[WFLY-14113] Move wildfly-preview to jakarta.authentication-api

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.redhat.com/browse/WFLY-14113

### CODE REVIEW

**Comment by bstansberry** [comment]
@scottmarlow Any objection?

File: ee-9/feature-pack/src/license/feature-pack-licenses.xml [hunk 1]
```diff
@@ -28,6 +28,22 @@
         </license>
       </licenses>
     </dependency>
+    <dependency>
+      <groupId>jakarta.authentication</groupId>
+      <artifactId>jakarta.authentication-api</artifactId>
+      <licenses>
+        <license>
+          <name>Apache License 2.0</name>
```
Comment (scottmarlow) [inline ee-9/feature-pack/src/license/feature-pack-licenses.xml:36]: I think this should be changed to Eclipse Public License - v 2.0 as per https://github.com/eclipse-ee4j/authentication/blob/master/LICENSE.md#eclipse-public-license---v-20

The below GPL v2 with classpath exception matches https://github.com/eclipse-ee4j/authentication/blob/master/LICENSE.md#the-gnu-general-public-license-gpl-version-2-june-1991 + https://github.com/eclipse-ee4j/authentication/blob/master/LICENSE.md#classpath-exception

File: ee-9/feature-pack/pom.xml [hunk 2]
```diff
@@ -378,7 +378,7 @@
         <dependency>
             <groupId>jakarta.interceptor</groupId>
             <artifactId>jakarta.interceptor-api</artifactId>
-            <version>${version.jakarta.interceptor.jakarata-interceptors-api}</version>
+            <version>${version.jakarta.interceptor.jakarta-interceptors-api}</version>
```
Comment (scottmarlow) [inline ee-9/feature-pack/pom.xml:381]: +1

File: ee-9/feature-pack/pom.xml [hunk 3]
```diff
@@ -394,9 +394,9 @@
             <scope>provided</scope>
         </dependency>-->
         <dependency>
-            <groupId>jakarta.security.auth.message</groupId>
-            <artifactId>jakarta.security.auth.message-api</artifactId>
-            <version>${version.jakarta.security.auth.message.jakarta-security-auth-message-api}</version>
+            <groupId>jakarta.authentication</groupId>
+            <artifactId>jakarta.authentication-api</artifactId>
```
Comment (scottmarlow) [inline ee-9/feature-pack/pom.xml:398]: :+1:

File: ee-9/feature-pack/src/license/feature-pack-licenses.xml (hunk 1 above)
Comment (bstansberry) [reply ee-9/feature-pack/src/license/feature-pack-licenses.xml]: @scottmarlow thanks; fixed

**Comment by bstansberry** [comment]
@scottmarlow Thanks. I've fixed the license problem; good catch!

**Review by scottmarlow — APPROVED** [review]

**Pull request merged**